src/sat_cli.py

```diff
@@ -34,7 +34,8 @@ def cli():
         try:
             cmd = input("$ ")
             server, port, toSend = gs.getInput(cmd, inVal="{}.cli.send_cmd({},{})".format(gs.satellite, len(cmd), cmd))
-            gs.transaction(server, port, toSend)
+            ret = gs.transaction(server, port, toSend)
+            print(ret)
```

Comment:
Do you want to say that it's a response code, or interpret the error code?

Reply:
What do you mean? If there is an error, the sat_cli creates its own error string to send back which would be printed here

Reply:
Sorry, I misread "data[2:]" as a number.